tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -138,7 +138,7 @@ def test_update_partial(self, cognite_client, new_transformation):
         )
 
     def test_list(self, cognite_client, new_transformation):
-        retrieved_transformations = cognite_client.transformations.list(limit=None)
+        retrieved_transformations = cognite_client.transformations.list(limit=None, destination_type="assets")
```

Comment:
necessary to filter by `destination_type` because the tenant has some `data model instance` transformations, which is not yet recognized as a type in this version. which will be added with https://github.com/cognitedata/cognite-sdk-python/pull/900.